### PR TITLE
MSC_FAT_VIEW Add config supprt

### PR DESCRIPTION
**Pull request by kasjer**
Code adds `CONFIG.TXT` file to FAT view.
File contains values available via conf API

### CODE REVIEW

File: hw/usb/tinyusb/msc_fat_view/src/entry_config.c [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+}
+
+OSTREAM_DEF(config_export);
+
+static struct config_export_stream export_stream = {
+    .out_stream.vft = &config_export_vft,
+};
+
+static void
+config_text_export(char *name, char *val)
```
Comment (sjanc) [inline hw/usb/tinyusb/msc_fat_view/src/entry_config.c:77]: nitpick could be constified

File: hw/usb/tinyusb/msc_fat_view/src/entry_config.c [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+{
+    int name_len = strlen(name);
+    int val_len = 0;
+    if (val) {
```
Comment (sjanc) [inline hw/usb/tinyusb/msc_fat_view/src/entry_config.c:81]: empty line

**Review by sjanc — APPROVED** [review]

File: hw/usb/tinyusb/msc_fat_view/src/entry_config.c (hunk 2 above)
Comment (kasjer) [reply hw/usb/tinyusb/msc_fat_view/src/entry_config.c:82]: fixed

File: hw/usb/tinyusb/msc_fat_view/src/entry_config.c (hunk 1 above)
Comment (kasjer) [reply hw/usb/tinyusb/msc_fat_view/src/entry_config.c:77]: Function prototype matches typedef from config/config.h
```c
/**
 * Called per-configuration variable being exported.
 *
 * @param name The name of the variable to export
 * @param val  The value of the variable to export
 */
typedef void (*conf_export_func_t)(char *name, char *val);```

**Pull request merged**